src/transform/ReferenceCollection.js

```diff
@@ -0,0 +1,253 @@
+import ElementUtilities from './ElementUtilities'
+import NodeUtilities from './NodeUtilities'
+import Polyfill from './Polyfill'
+
+const REFERENCE_SELECTOR = '.reference'
+
+/**
+ * Is Citation.
+ * @param  {!string}  href
+ * @return {!boolean}
+ */
+const isCitation = href => href.indexOf('#cite_note') > -1
```

Comment:
I think I prefer to keep the method narrowly scoped vs passing it the entire anchor element. Open to refactoring this later if you feel strongly about it though :)

I added and used a constant per your suggestion. For using the anchor rather than the `href` string